### PR TITLE
fix: script fixes

### PR DESCRIPTION
**Pull request by kelemeno**
# What ❔

## Checklist

- [ ] PR title corresponds to the body of PR (we generate changelog entries from PRs).
- [ ] Tests for the changes have been added / updated.
- [ ] Documentation comments have been added / updated.

### CODE REVIEW

File: l1-contracts/src.ts/deploy.ts [hunk 1]
```diff
@@ -124,7 +125,7 @@ export class Deployer {
       : hexlify(hashL2Bytecode(readSystemContractsBytecode("DefaultAccount")));
     this.ownerAddress = config.ownerAddress != null ? config.ownerAddress : this.deployWallet.address;
     this.chainId = parseInt(process.env.CHAIN_ETH_ZKSYNC_NETWORK_ID!);
-    this.l1ChainId = parseInt(config.l1ChainId);
+    this.l1ChainId = parseInt(config.l1ChainId || getNumberFromEnv("ETH_CLIENT_CHAIN_ID"));
```
Comment (StanislavBreadless) [inline l1-contracts/src.ts/deploy.ts:128]: we have a method `getL1ChainId()`

File: l2-contracts/src/deploy-shared-bridge-on-l2-through-l1.ts [hunk 2]
```diff
@@ -11,49 +10,9 @@ import { GAS_MULTIPLIER } from "../../l1-contracts/scripts/utils";
 import * as hre from "hardhat";
 import { L2_ASSET_ROUTER_ADDRESS, L2_NATIVE_TOKEN_VAULT_ADDRESS } from "../../l1-contracts/src.ts/utils";
 
-export const L2_SHARED_BRIDGE_ABI = hre.artifacts.readArtifactSync("L2AssetRouter").abi;
+// export const L2_SHARED_BRIDGE_ABI = hre.artifacts.readArtifactSync("L2AssetRouter").abi;
```
Comment (StanislavBreadless) [inline l2-contracts/src/deploy-shared-bridge-on-l2-through-l1.ts:13]: why commented out code?

File: l2-contracts/src/deploy-shared-bridge-on-l2-through-l1.ts [hunk 3]
```diff
@@ -68,7 +27,7 @@ async function main() {
     .option("--erc20-bridge <erc20-bridge>")
     .option("--skip-initialize-chain-governance <skip-initialize-chain-governance>")
     .action(async (cmd) => {
-      const chainId: string = cmd.chainId ? cmd.chainId : process.env.CHAIN_ETH_ZKSYNC_NETWORK_ID;
+      // const chainId: string = cmd.chainId ? cmd.chainId : process.env.CHAIN_ETH_ZKSYNC_NETWORK_ID;
```
Comment (StanislavBreadless) [inline l2-contracts/src/deploy-shared-bridge-on-l2-through-l1.ts:30]: same here

File: l1-contracts/src.ts/deploy.ts (hunk 1 above)
Comment (kelemeno) [reply l1-contracts/src.ts/deploy.ts]: its async, this is the constructor

**Pull request merged**